project/V.scala

```diff
@@ -6,6 +6,7 @@ object V {
   val scala212 = "2.12.17"
   val scala213 = "2.13.10"
   val scala3 = "3.2.2"
+  val scala3RC = "3.3.0-RC1"
```

Comment:
Maybe we could have:
```suggestion
  val scala3RC = Option("3.3.0-RC1")
```
which would avoid issues with removing/readding it?

Reply:
Ah good idea, I'll add this.